src/wallet/walletdb.h

```diff
@@ -247,6 +245,7 @@ class CWalletDB
     bool WriteVersion(int nVersion);
 private:
     CDB batch;
+    CWalletDBWrapper& wrapper;
```

Comment:
In commit "CWalletDB: Store the update counter per wallet"

Might be nice to call this member dbw instead of wrapper for consistency with the wallet member.